include/osw_hal.h

```diff
@@ -6,6 +6,7 @@
 #include <gfx_2d_print.h>
 
 #include <string>
```

Comment:
Could you try to use the Ardunio String library? It can be (always) used via `String` and does not require you to include the `std::string` lib.

Reply:
sure this wasn't me btw this is already in the development branch.

Reply:
Okay that's on @rawsignal !